Tensile/SolutionStructs.py

```diff
@@ -2401,22 +2401,26 @@ def assignDerivedParameters(state):
         state['_'+s] = state[s]
         #del state[s]
 
-    dataType = state["ProblemType"]["DataType"]
-    state["_GlobalAccumulation"] = None
-    if ((dataType.isBFloat16() or dataType.isHalf())
-        and state["ProblemType"]["HighPrecisionAccumulate"]
-        and state["GlobalSplitU"] > 1):
-      if state["GlobalSplitUAlgorithm"] == "SingleBuffer":
-        state["_GlobalAccumulation"] = 'SingleBuffer'
-      if state["GlobalSplitUAlgorithm"] == "MultipleBuffer":
-        state["_GlobalAccumulation"] = 'MultipleBuffer'
-
-    if state["_GlobalAccumulation"] == 'SingleBuffer':
-      state["_WorkspaceSizePerElemC"] = 4
-    elif state["_GlobalAccumulation"] == 'MultipleBuffer':
-      state["_WorkspaceSizePerElemC"] = 4 * state["GlobalSplitU"]
-    else:
+    if ("_GlobalAccumulation" not in state) or ("_WorkspaceSizePerElemC" not in state):
+      state["_GlobalAccumulation"] = None
       state["_WorkspaceSizePerElemC"] = 0
+      if state["GlobalSplitU"] > 1:
+        computeName  = state["ProblemType"]["ComputeDataType"].toName()
+        computeBytes = state["ProblemType"]["ComputeDataType"].numBytes()
+        if state["ProblemType"]["DataType"].isHalf() and state["ProblemType"]["HighPrecisionAccumulate"]:
+          computeName  = 'single'
+          computeBytes = 4
+
```

Comment:
No need to consider bf16 and in8 here and everywhere else?

Reply:
they have correct compute type
Only half is wrong on compute type.

We should correct compute type of hh to s, but all code will get influence.
So we don't correct.